packages/app/src/runner/ResizablePanels.vue

```diff
@@ -1,9 +1,10 @@
 <template>
   <div
-    class="flex overflow-y-hidden"
+    id="resizable-panels-root"
+    class="flex"
     :class="{
       'select-none': panel1IsDragging || panel2IsDragging,
-      'overflow-x-auto': !isFirefox
+      'overflow-x-hidden': isFirefox
```

Comment:
this lets the `!important` that comes with the utility class override the otherwise-preferred overflow.